src/utils/filter-utils.js

```diff
@@ -123,7 +146,10 @@ export function getDefaultFilter(dataId) {
     // plot
     plotType: PLOT_TYPES.histogram,
     yAxis: null,
-    interval: null
+    interval: null,
+
+    // polygon:
+    layerId: null
```

Comment:
I think it's time we create unique properties for each filter. instead of adding everything to default filter.  I imagine we can store not only layerId, but also layer.columns, and converted polygon geometry by turf in it.